person.rb

```diff
@@ -1,12 +1,13 @@
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
```

Comment:
- Great job for adding Getters  `rentals`, I think you forget to add setter so please define the rentals as 
`attr_accessor :rentals` and add a method for adding rentals to create the new rental achieving the 'has-many' association as previous reviewer suggested 👍 
```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
@binhussen Done👍!!!